content/en/account_management/audit_logs.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Audit Logs
+kind: documentation
+further_reading:
+- link: "/account_management/org_settings/"
+  tag: "Documentation"
+  text: "Learn more about organization settings"
+---
+
+<div class="alert alert-warning">This feature is in public beta. Contact <a href="https://www.datadoghq.com/support/">Datadog support</a> for more information.</div>
+
+## Overview
+
+As an administrator or security team member, you can use [Audit Logs][1] to see who is using Datadog within your organization and the context in which they are using Datadog. As an individual, you can see a stream of your own actions, too.
+
+There are two types of events that can occur within an audit log:
+
+1. Request events, which translate all requests made to Datadog’s API into customer records.
+2. Product-specific events, such as:
+    - Changes within your organization or account. For example, if a user changes a dashboard, you are able to see who made the last change and at what time.
```

Comment:
The most common examples we hear on Log Management are:
1. I want to know when someone changed the retention of an index
2. I want to know who modified the processing/pipelines and when.
3. I want to know who modified an exclusion filters.
These are common billing impact and access impact questions that Ali and I come across most often. They will resonate more than the dashboard  and breaking events example